### ISSUE TITLE
[OUDS] Misplaced `.was-validated` selector on control items

### ISSUE THREAD

**Issue by hannahiss** [Orange-OpenSource/Orange-Boosted-Bootstrap#3349]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Error messages and negative colors should be applied to `:invalid` elements when any parent has `.was-validated` class (typically the form).

Currently, color is not applied to divider, global `control-item-error-message` does not appear and icon appears on the right side with wrong paddings, because it is made to have `.was-validated` directly on control-item container or list. 

What we should have (screen copy with `.is-invalid` class and no `.was-validated`):

<img width="707" height="818" alt="Image" src="https://github.com/user-attachments/assets/d5390899-3e76-429d-9c55-394eb0c8ab71" />

What we have (when we don't have the `.is-invalid` class, but we have `.was-validated` on the form and `:invalid` on the inputs) - it should the same than above: 

<img width="700" height="682" alt="Image" src="https://github.com/user-attachments/assets/4b17ebd7-b865-4ed0-a7b8-c7a4aaab862f" />

### Reduced test cases

Set `.was-validated` on the `<form>` and make the control-item input `:invalid` (without the `.is-invalid` class).

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome

### What version of Boosted are you using?

v1.0.0 OUDS
